### PR TITLE
Add basic documentation for Druid connector

### PR DESCRIPTION
**Pull request by electrum**

### CODE REVIEW

File: presto-docs/src/main/sphinx/connector/druid.rst [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+.. contents::
+    :local:
+    :backlinks: none
+    :depth: 1
+
+Overview
+--------
+
+The Druid Connector allows querying `Apache Druid <https://druid.apache.org/>`_ data from Presto.
```
Comment (mosabua) [inline presto-docs/src/main/sphinx/connector/druid.rst:13]: Should be above contents list and then drop Overview section title..

File: presto-docs/src/main/sphinx/connector/druid.rst [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+Configuration
+-------------
+
+To configure the Druid connector, create a catalog properties file
+``etc/catalog/druid.properties`` with the following contents, replacing
```
Comment (mosabua) [inline presto-docs/src/main/sphinx/connector/druid.rst:19]: somehow need to express that this is just an example name ..

Comment (electrum) [reply presto-docs/src/main/sphinx/connector/druid.rst:19]: Agreed, but this needs to be fixed across all connectors

File: presto-docs/src/main/sphinx/connector/druid.rst (hunk 1 above)
Comment (electrum) [reply presto-docs/src/main/sphinx/connector/druid.rst:13]: Done

**Pull request merged**